src/test/resources/config/application.yml

```diff
@@ -149,6 +149,10 @@ management:
         env:
             enabled: true
 
+spring-hibernate-query-utils:
+    n-plus-one-queries-detection:
```

Comment:
This sounds like a good thing to have, can you shortly explain why you disabled it? 🙂

Reply:
Indeed it does sound nice, but I disabled it because the builds were indefinitely pending since too much memory was being used and apparently this automatic detection was causing it.

Either way I'm not sure if this was even working since I didn't test it, and we can look at re-enabling this in the future.

Reply:
can you please add a comment with this explanation to the file? otherwise, other developers will be confused. It's also not clear that it is disabled

Reply:
Yes, the memorry issue arose after adding the library. I disable the detection of n-plus-one-queries because it's a feature of the library, and I just added a comment to make it clear what the property is and why it's disabled.